op/model.py

```diff
@@ -194,26 +207,34 @@ def __getitem__(self, key):
 
 # We mix in MutableMapping here to get some convenience implementations, but whether it's actually
 # mutable or not is controlled by the flag.
-class RelationUnitData(LazyMapping, MutableMapping):
-    def __init__(self, relation, unit, is_mutable, backend):
+class RelationEntityData(LazyMapping, MutableMapping):
```

Comment:
I suggest we name this as _RelationDataContent_ and amke the _entity_ attribute below private. We encourage this to be seen as content inside the "relation data", instead of the currently prevalent idea of independent "unit data" or "application data".